autospec/build.py

```diff
@@ -198,11 +198,8 @@ def get_uniqueext(dirn, dist, name):
 
 def set_mock():
     global mock_cmd
-    # get group list of current user
-    user_grps = [grp.getgrgid(g).gr_name for g in os.getgroups()]
-    if os.path.exists('/usr/bin/mock'):
-        if 'mock' not in user_grps:
-            mock_cmd = 'sudo /usr/bin/mock'
+    if os.path.basename(os.path.realpath('/usr/bin/mock')) != "consolehelper":
```

Comment:
I think this is a good idea though I'm inclined to say the test should be if it is mock instead of not consolehelper.

Reply:
Changed to check for 'mock', and added a short note about consolehelper.
Fixed the unused import and re-pushed.